peer/brontide.go

```diff
@@ -3157,3 +3170,19 @@ func (p *Brontide) BytesReceived() uint64 {
 func (p *Brontide) BytesSent() uint64 {
 	return atomic.LoadUint64(&p.bytesSent)
 }
+
+// LastRemotePingPayload returns the last payload the remote party sent as part
+// of their ping.
+func (p *Brontide) LastRemotePingPayload() []byte {
+	pingPayload := p.lastPingPayload.Load()
+	if pingPayload == nil {
+		return []byte{}
+	}
+
+	pingBytes, ok := pingPayload.([]byte)
```

Comment:
Did some local regtest tests and this seems to need to be `pingBytes, ok := pingPayload.(lnwire.PingPayload)` in order for `ok` to be true.